MdeModulePkg/Universal/Console/ConPlatformDxe/ConPlatform.c

```diff
@@ -808,10 +808,15 @@ MatchUsbClass (
   DeviceClass    = DevDesc.DeviceClass;
   DeviceSubClass = DevDesc.DeviceSubClass;
   DeviceProtocol = DevDesc.DeviceProtocol;
-  if (DeviceClass == 0) {
+
+  // MSCHANGE - BEGIN - Add support for IAD-type multi-function devices.

```

Comment:
Recommend (but feel free to dismiss):

```suggestion
  // MU_CHANGE - BEGIN - Add support for IAD-type multi-function devices.

```

Reply:
good suggestion, Updated it.